### PR TITLE
📦 NEW: Add US States #6 via @bernardodsanderson 

### PR DESCRIPTION
**Pull request by bernardodsanderson**
@ahmadawais - This PR adds states as a view option. I tried to follow your style but let me know if you'd like any changes or updates.

### CODE REVIEW

File: utils/table.js [hunk 1]
```diff
@@ -37,6 +37,24 @@ module.exports = {
 		`critical`,
 		`per-million`
 	],
+	singleStates: [
+		`Country`,
```
Comment (ahmadawais) [inline utils/table.js]: You meant state

Comment (bernardodsanderson) [reply utils/table.js]: Oops, correct, fixed it.

**Comment by ahmadawais** [comment]
I see it is working fine but sort is not working.

**Comment by ahmadawais** [comment]
Thank you @bernardodsanderson I refactored the code you wrote a lot, in case you want to check it out. Also added the sort feature set.

🔗 https://github.com/ahmadawais/corona-cli/releases/tag/2.5.0
🔗 https://www.npmjs.com/package/corona-cli

**Pull request merged**